### PR TITLE
@joeyAghion => [Image Uploading] Drop redundant gemini config, consolidate on @artsy/gemup #migration

### PR DESCRIPTION
**Pull request by mzikherman**
https://artsyproduct.atlassian.net/secure/RapidBoard.jspa?rapidView=64&projectKey=PLATFORM&modal=detail&selectedIssue=PLATFORM-1457

This can unset the `GEMINI_S3_ACCESS_KEY` config post-deploy in the staging and prod environments.

(I tested both editing a JSON page and consignment flow to verify things were still working).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -129,7 +130,6 @@
     "artsy-backbone-mixins": "artsy/artsy-backbone-mixins",
     "artsy-eigen-web-association": "artsy/artsy-eigen-web-association",
     "artsy-ezel-components": "artsy/artsy-ezel-components",
-    "artsy-gemini-upload": "0.0.6",
```
Comment (mzikherman) [inline package.json]: lol turns out there's [yet another library](https://github.com/artsy/artsy-gemini-upload) with a slightly different API...this one was a jQuery extension.

File: src/desktop/apps/consign/client/actions.js [hunk 2]
```diff
@@ -334,7 +334,6 @@ export function handleImageUpload(file) {
         const options = {
           acl: "private",
           app: sd.CONVECTION_GEMINI_APP,
-          key: sd.GEMINI_S3_ACCESS_KEY,
```
Comment (mzikherman) [inline src/desktop/apps/consign/client/actions.js]: 👋 this is what the changes are like for clients using the correct `@artsy/gemup`

**Comment by mzikherman** [comment]
Seems like spurious failures, but I'll retry.

File: package.json (hunk 1 above)
Comment (joeyAghion) [reply package.json]: This was maybe the last use of this library, according to [my search](https://github.com/search?q=org%3Aartsy+artsy-gemini-upload&type=Code). Can we mark that library as deprecated, archive the repo, and update the project list?

**Comment by joeyAghion** [comment]
👍 from me once the conflicts are resolved.

I've added `#migration` to the title so we remember to unset the config once deployed.

File: package.json (hunk 1 above)
Comment (mzikherman) [reply package.json]: Definitely!

**Pull request merged**